index.html

```diff
@@ -232,17 +231,17 @@ <h1>Get Started</h1>
         I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
       </p>
```

Comment:
- [ ] As the previous reviewer suggested, You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using the templates in Figma. Kindly try to match the Figma design. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![getyourfigma](https://github.com/donice/Portfolio-Microverse/assets/70483835/d7742489-fcc7-47c1-a4ff-ae9a77ffb33f)|  ![getfigma](https://github.com/donice/Portfolio-Microverse/assets/70483835/b9ababb7-cb74-4a87-adb2-1cbaf2d6130d)